iOS/MMIWG2S/ViewControllers/ViewController1.swift

```diff
@@ -0,0 +1,163 @@
+        gradientView.contentMode = .scaleAspectFill
+        gradientView.clipsToBounds = true
+        view.addSubview(gradientView)
+        gradientView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            gradientView.topAnchor.constraint(equalTo: view.topAnchor),
+            gradientView.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 1),
+            gradientView.heightAnchor.constraint(equalTo: view.heightAnchor),
+            gradientView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            ])
+
+        setupUI()
+    }
+    
+    func setupUI() {
+        
+        //Setup UI
+        let labelMMIW = UILabel(frame:CGRect(x:0,y:0,width:50,height:21))
+        let labelMMIWInfo = UILabel(frame:CGRect(x:0,y:0,width:200,height:200))
+        let buttonNext = UIButton(frame:CGRect(x:0,y:0, width:30, height:17))
+         
+        labelMMIW.translatesAutoresizingMaskIntoConstraints = false
+        labelMMIWInfo.translatesAutoresizingMaskIntoConstraints = false
+        buttonNext.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(labelMMIW)
+        view.addSubview(labelMMIWInfo)
+
+        pc1View.translatesAutoresizingMaskIntoConstraints = false
+        pc2View.translatesAutoresizingMaskIntoConstraints = false
+        pc3View.translatesAutoresizingMaskIntoConstraints = false
+        pc4View.translatesAutoresizingMaskIntoConstraints = false
+    
+        labelMMIW.textAlignment = .center
+        labelMMIW.text = NSLocalizedString("slideviewheader", comment: "Text")
+        labelMMIW.textColor = UIColor.white
+        labelMMIW.font = UIFont(name: "Roboto-Regular", size:36.0 )
+
+        labelMMIWInfo.numberOfLines = 0
+        labelMMIWInfo.textColor = UIColor.white
+        labelMMIWInfo.font = UIFont(name: "Roboto-Regular", size:25.0 )
+        labelMMIWInfo.text = NSLocalizedString("slideview1text", comment: "text")
+        
+        pc1View.backgroundColor = UIColor.red
+        pc2View.backgroundColor = UIColor.white
+        pc3View.backgroundColor = UIColor.white
+        pc4View.backgroundColor = UIColor.white
+
+        buttonNext.alpha = 1.0
+        buttonNext.setTitleColor(UIColor.white, for:.normal)
+        buttonNext.setTitleColor(UIColor.green, for:.selected)
+
+        buttonNext.imageEdgeInsets = UIEdgeInsets(top:0, left:0, bottom:0, right:0 )
+        buttonNext.contentEdgeInsets = UIEdgeInsets(top:0, left:0, bottom:0, right:0 )
+        buttonNext.titleLabel?.font = UIFont.systemFont(ofSize: 17)
+        buttonNext.setTitle("Next", for:.normal)
+        
+
+        buttonNext.addTarget(self, action: #selector(self.changePage(sender:)), for: .touchUpInside)
+        
+        let bottomView = UIView(frame:CGRect(x:0, y:0, width:100, height:50))
+        bottomView.addSubview(pc1View)
+        bottomView.addSubview(pc2View)
+        bottomView.addSubview(pc3View)
+        bottomView.addSubview(pc4View)
```

Comment:
With these in an array it would likely look `pc1views.forEach { bottomView.addSubview($0) }`